Assets/Mirage/Editor/WelcomeWindow/WelcomeWindow.cs

```diff
@@ -58,6 +60,8 @@ public class WelcomeWindow : EditorWindow
         private static string firstStartUpKey = string.Empty;
         private const string firstTimeMirageKey = "MirageWelcome";
 
+        private const string changeLogPath = "Packages/com.miragenet.mirage/CHANGELOG.md";
```

Comment:
hardcoded path is an issue,  this does not work if you import it directly without UPM.

What we could do is write the latest version changelog to a file in the resources folder instead.